hathor/transaction/storage/transaction_storage.py

```diff
@@ -409,24 +418,35 @@ def update_block_height_cache_new_chain(self, height: int, block: Block) -> None
 
         block_height = height
         side_chain_block = block
-        add_to_cache: List[Tuple[int, bytes]] = []
+        add_to_cache: List[Tuple[int, bytes, int]] = []
```

Comment:
Should we replace the tuple by a namedtuple? If I got it right, it should be `(height, blk_id, timestamp)`.

Reply:
Makes sense. I'll do that.

Reply:
Done.

Reply:
Great! Thanks!